lldb/include/lldb/Target/Statistics.h

```diff
@@ -122,6 +122,7 @@ struct ModuleStats {
   double symtab_index_time = 0.0;
   double debug_parse_time = 0.0;
   double debug_index_time = 0.0;
+  llvm::StringMap<double> symbol_locator_time;
```

Comment:
We should move `StatisticsMap` into this file and then have `StatisticsMap` use a `llvm::StringMap<double>` instead of a `std::unordered_map<std::string, double> map;`. Then you can just copy it over into this object.